recipes/ephyviewer/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ephyviewer" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "47a4bf49e6cbfb605caf1b835a9d5a6b91c30a103249744aab5e776535ca9787" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - ephyviewer=ephyviewer.scripts:launch_standalone_ephyviewer
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
I can try that. I see that those flags are not present in ``example.yaml``, but I've also seen in the docs (e.g. [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip)) that ``--no-deps`` should be used. I think ``conda skeleton`` also put the flags in when I tried it. Was there a policy change, and the docs are out of date? Unfortunately [git blame](https://github.com/conda-forge/staged-recipes/blame/94aba5d772f68d7945453b68ffd32641bcb9e266/recipes/example/meta.yaml#L33) is not very useful for tracking down the reason for that change in ``example.yaml``.